cypress/integration/catalog/categories.js

```diff
@@ -185,28 +196,40 @@ filterTests({ definedTags: ["all"] }, () => {
       let firstCategory;
       let secondCategory;
 
-      createCategory({ name: firstCategoryName }).then(categoryResp => {
-        firstCategory = categoryResp;
-      });
-      createCategory({ name: secondCategoryName }).then(categoryResp => {
-        secondCategory = categoryResp;
-        cy.visit(urlList.categories)
-          .searchInTable(startsWith)
-          .get(categoryRow(firstCategory.id))
-          .find(BUTTON_SELECTORS.checkbox)
-          .click()
-          .get(categoryRow(secondCategory.id))
-          .find(BUTTON_SELECTORS.checkbox)
-          .click()
-          .get(BUTTON_SELECTORS.deleteIcon)
-          .click()
-          .addAliasToGraphRequest("CategoryBulkDelete")
-          .get(BUTTON_SELECTORS.submit)
-          .click()
-          .waitForRequestAndCheckIfNoErrors("@CategoryBulkDelete");
-        getCategory(firstCategory.id).should("be.null");
-        getCategory(secondCategory.id).should("be.null");
-      });
+      visitAddPage();
+      createCategory({
+        name: firstCategoryName,
+        description: firstCategoryName
+      })
+        .its("response.body.data.categoryCreate.category")
+        .then(categoryResp => {
+          firstCategory = categoryResp;
+        });
+      visitAddPage();
+      createCategory({
+        name: secondCategoryName,
+        description: secondCategoryName
+      })
+        .its("response.body.data.categoryCreate.category")
+        .then(categoryResp => {
+          secondCategory = categoryResp;
+          cy.visit(urlList.categories)
+            .searchInTable(startsWith)
+            .get(categoryRow(firstCategory.id))
+            .find(BUTTON_SELECTORS.checkbox)
+            .click()
+            .get(categoryRow(secondCategory.id))
+            .find(BUTTON_SELECTORS.checkbox)
+            .click()
+            .get(BUTTON_SELECTORS.deleteIcon)
+            .click()
+            .addAliasToGraphRequest("CategoryBulkDelete")
```

Comment:
This test is failing - it is too fast ;)
![image](https://user-images.githubusercontent.com/29308910/163346022-29314063-c3de-4b95-8058-820fdc669d92.png)

Reply:
this issue will be solved under https://app.clickup.com/t/2549495/SALEOR-6560